vkcs/monitoring_channel.go

```diff
@@ -0,0 +1,89 @@
+package vkcs
+
+import (
+	"github.com/gophercloud/gophercloud"
+	"net/http"
+)
+
+func instanceChannelURL(c ContainerClient, pid string, id string) string {
+	return c.ServiceURL(pid, "notification_channels", id)
+}
+
+func createChannelURL(c ContainerClient, pid string) string {
+	return c.ServiceURL(pid, "notification_channels")
+}
+
+type commonChannelResult struct {
+	gophercloud.Result
+}
+
+// extract is used to extract result into short response struct
+func (r commonChannelResult) extract() (*ChannelOut, error) {
+	var c *ChannelOut
+	if err := r.ExtractInto(&c); err != nil {
+		return nil, err
+	}
+	return c, nil
+}
+
+type deleteChannelResult struct {
+	gophercloud.Result
+}
+
+func (r deleteChannelResult) extractErr() error {
+	return r.Err
+}
```

Comment:
@ftersin @schirevko 
In public DNS I decided to use capitalized names for result methods, because it's more convenient and it would be easier to move client logic into separate package, which I believe should be done. If we stick to this way, you could just use gophercloud.ErrResult for deleteChannelResult.